src/types/IJaaSMeetingProps.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import IJitsiMeetingOptions from "./IJitsiMeetingOptions";
+import { JaaSMeeting } from "..";
+
+/**
+ * The type of the React {@code Component} props of {@link JaaSMeeting}.
+ */
+export default interface IJaaSMeetingProps {
+  /**
+   * The id of the component
+   */
+  id: string;
+
+  /**
+   * The App ID that provides an isolated context and prefixes the room name
+   */
+  appId: string;
+
+  /**
+   * The optional arguments for the conference
+   */
+  options: IJitsiMeetingOptions;
```

Comment:
I'd say we flatten the structure here and have all options be top level options.